### PR TITLE
Tab

### PR DESCRIPTION
**Pull request by edsonsnunes**

### CODE REVIEW

File: projects/ion/src/lib/tab/tab.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+
+export type Size = 'sm' | 'md' | 'lg';
```
Comment (iurynogueira) [inline projects/ion/src/lib/tab/tab.component.ts:3]:
```suggestion
export type TabSize = 'sm' | 'md' | 'lg';
```

change this not to confuse with chip size.

**Review by iurynogueira — CHANGES_REQUESTED** [review]
Nice work! Don't forget of add stories.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,13 @@ Component library following a great Design System
 
 <a href="https://codeclimate.com/github/iurynogueira/ion/maintainability"><img src="https://api.codeclimate.com/v1/badges/22e2c6dab72fd7234e24/maintainability" /></a>
 <a href="https://codeclimate.com/github/iurynogueira/ion/test_coverage"><img src="https://api.codeclimate.com/v1/badges/22e2c6dab72fd7234e24/test_coverage" /></a>
+<<<<<<< HEAD
+[![Automated Tests and Linting](https://github.com/iurynogueira/ion/actions/workflows/main.yml/badge.svg)](https://github.com/iurynogueira/ion/actions/workflows/main.yml)
+=======
 [![Production](https://github.com/iurynogueira/ion/actions/workflows/prod.yml/badge.svg?branch=main)](https://github.com/iurynogueira/ion/actions/workflows/prod.yml)
 
+> > > > > > > c497977328053cfb30bfc66a488ec7b3f1f1d35c
```
Comment (iurynogueira) [inline README.md:12]: Fix conflict

File: projects/ion/src/lib/tab/tab.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+describe('TabComponent', () => {
+  it.each(['about', 'profile'])(
+    'should render label %s',
+    async (tabName: string) => {
+      await sut({ label: tabName });
+      expect(screen.getAllByText(tabName)).toHaveLength(1);
+    }
+  );
+
+  it.each(['md', 'lg'])(
```
Comment (iurynogueira) [inline projects/ion/src/lib/tab/tab.component.spec.ts:28]:
```suggestion
  it.each(['sm', 'md', 'lg'])(
```

**Review by iurynogueira — CHANGES_REQUESTED** [review]

**Review by iurynogueira — APPROVED** [review]

File: projects/ion/src/lib/tab/tab.component.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  selector: 'ion-tab',
+  templateUrl: './tab.component.html',
+  styleUrls: ['./tab.component.scss'],
+})
+export class TabComponent {
+  @Input() label!: string;
+  @Input('tabSize') tabSize?: TabSize = 'sm';
+  @Input('disabled') disabled?: boolean;
+  @Input('selected') selected?: boolean;
+  @Input('direction') direction?: Direction = 'bottom';
```
Comment (iurynogueira) [inline projects/ion/src/lib/tab/tab.component.ts:16]:
```suggestion
  @Input() tabSize?: TabSize = 'sm';
  @Input() disabled?: boolean;
  @Input() selected?: boolean;
  @Input() direction?: Direction = 'bottom';
```

**Review by iurynogueira — APPROVED** [review]

File: projects/ion/src/lib/tab/tab.component.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  [ngClass]="{
+    tab: !selected,
+    'tab-selected': selected,
+    'tab-sm': tabSize === 'sm',
+    'tab-md': tabSize === 'md',
+    'tab-lg': tabSize === 'lg'
+  }"
+  [class]="'border-' + direction"
+>
+  {{ label }}
```
Comment (MarcosOliveir4) [inline projects/ion/src/lib/tab/tab.component.html:13]: in this situation the label will not give any error if you do not pass the label through the attribute.
there are two ways to adjust this

1° in the html file
`{{ label || 'Configure uma label' }}`

2° in the ts file
`@Input() label = "Configure uma label"`

**Review by MarcosOliveir4 — CHANGES_REQUESTED** [review]

**Review by Deeved — APPROVED** [review]

**Pull request merged**